tests/github/interactive_api_director/experiments/pytorch_kvasir_unet/experiment.py

```diff
@@ -25,7 +25,7 @@ def run():
         tls=False
     )
 
-    shard_registry = federation.get_shard_registry()
```

Comment:
Function actually returns a value. If this was changed because of the unused `shard_variable`, the line itself can be removed.

Reply:
I don't see `shard_registry` used elsewhere. +1 to removing.